scripts/devaccess/helper_scripts/rhmi_3scale_dump.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+THREESCALE_DUMP_SCRIPT=3scale-dump.sh
```

Comment:
To avoid any potential issues if the script's location is not in $PATH when run through the devaccess wrapper:

```suggestion
THREESCALE_DUMP_SCRIPT=/usr/local/bin/3scale-dump.sh
```